source/blogs/dbc/dbc.rst

```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
 a debug device by using xDBC on the debug target. The debug device presented by the
 debug target can be used by debug host for low level system debugging of target.
 
-Github PR's link for enabling adb over DbC for Celadon KBL NUC
---------------------------------------------------------------
+Github PRs for enabling adb over DbC for |C| KBL NUC
```

Comment:
Capitalize "hub" / GitHub 
Suggested rewrite: 
GitHub PRs to enable adb over DbC for |C| KBL NUC